### PR TITLE
Fix sphinx warning and link inside the rips sphinx doc

### PR DESCRIPTION
**Pull request by VincentRouvreau**

### CODE REVIEW

File: src/python/doc/rips_complex_user.rst [hunk 1]
```diff
@@ -400,12 +400,10 @@ The output is:
 
     [(0, (3.1622776601683795, inf)), (0, (3.1622776601683795, 5.39834563766817)), (0, (3.1622776601683795, 5.39834563766817))]
 
-.. _dtm-rips-complex:
-
 DTM Rips Complex
 ----------------
 
-`DTMRipsComplex <rips_complex_ref.html#dtm-rips-complex-reference-manual>`_ builds a simplicial complex from a point set or a full distance matrix (in the form of ndarray), as described in the above example.
+`DTMRipsComplex (reference manual) <rips_complex_ref.html#dtm-rips-complex-reference-manual>`_ builds a simplicial complex from a point set or a full distance matrix (in the form of ndarray), as described in the above example.
```
Comment (mglisse) [inline src/python/doc/rips_complex_user.rst:406]: I would have tried ``:class:`~gudhi.dtm_rips_complex.DTMRipsComplex` ``, but why not.

File: src/python/doc/rips_complex_user.rst [hunk 2]
```diff
@@ -378,7 +378,7 @@ Example from a point cloud combined with DistanceToMeasure
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Combining with DistanceToMeasure, one can compute the DTM-filtration of a point set, as in `this notebook <https://github.com/GUDHI/TDA-tutorial/blob/master/Tuto-GUDHI-DTM-filtrations.ipynb>`_. 
-Remark that `DTMRipsComplex <rips_complex_user.html#dtm-rips-complex>`_ class provides exactly this function.
+Remark that `DTM Rips Complex (user manual) <rips_complex_user.html#dtm-rips-complex>`_ class provides exactly this function.
```
Comment (mglisse) [inline src/python/doc/rips_complex_user.rst:381]: It is a bit confusing to say that it is a class, but put spaces in the middle of its name. Bah...

File: src/python/doc/rips_complex_user.rst (hunk 1 above)
Comment (VincentRouvreau) [reply src/python/doc/rips_complex_user.rst]: Oh yes I was not thinking of this trick. I used it on 83db295e

File: src/python/doc/rips_complex_user.rst (hunk 2 above)
Comment (VincentRouvreau) [reply src/python/doc/rips_complex_user.rst]: That's fair : 83db295e

**Pull request merged**